locales/en-US.json

```diff
@@ -252,7 +252,8 @@
   "title": "Title",
   "tooltips": {
     "changeLanguage": "Language",
-    "toggleDarkMode": "Toggle dark mode"
+    "darkMode": "Dark mode",
```

Comment:
What about "Switch to X mode"?

Reply:
I think it's not needed, I checked the first similar example I had at hand (Flood) and it only said "dark theme" "light theme"

Reply:
But that's really a preference haha

Reply:
Imo it's clearer and 100% explicit, without place to imagination.

If an user it's in dark mode and see then tooltip as "Light mode" he/she might wonder what mode it's currently in. For a lot of people this is obvious but maybe not so much for elderly or color blind :).

With switch there's no possible doubt.

Reply:
Done